data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	A series of charts are thrown up on the screen. You're not an expert at reading this sort of data but highlighted percentages ranging from as low as 4% to only as high as 39% do not look promising. The final graphic is a map of stages of what is labelled as 'artifact penetration'.`
+			`	"Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is other than the truth, at least not while still holding to the claim of being a representative government," explains Mr Yang.`
+			`	He checks the date, "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months or lose so much credibility as to compromise electoral stability, the actual time limit is probably sooner."`
+			choice
+				`	"Would that be so bad?"`
+			`	"It is always bad to be forced into acknowledging a secret, if we act sooner we may control the manner of that secret's release," says Yang.`
+			`	Sayari looks at you, "We will have to come up with a plan, and quickly, but for the moment may I ask a favour of the war hero in the room? Please meet up with the security forces converging on Allhome and help them route the pirates back to the other side of the wormhole. It will buy us some breathing space and some options."`
+			`	It is so hard to say "no" to a Hai and their cute squirrelly features.`
+			choice
+				`	"I'll do what I can."`
+					decline
+
+#If a republic/syndicate campaign gets added this on complete text will need variants.
+	
+	
+event "pirates respond to leaks"
+	#Add pirate spawns to Waypoint and Heia Due
+	system "Waypoint"
+		government "Pirate"
+		add fleet "Large Northern Pirates" 3000
+		add fleet "Small Northern Pirates" 1000
+		add fleet "Large Core Pirates" 3000
+		add fleet "Small Core Pirates" 1000
+		remove fleet "Small Hai"
+		remove fleet "Large Hai"
+	system "Heia Due"
+		add fleet "Marauder fleet IX" 3000
+		add fleet "Large Northern Pirates" 3000
+		add fleet "Large Core Pirates" 3000
+		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
+		
+	
+#If a republic/syndicate campaign gets added this text will need variants.
+mission "Hai Leaks Response 2A"
+	landing
+	invisible
+	source Deep
+	to offer
+		has "Hai Leaks Response 2: active"
+	on offer
+		log "People" "Alondo Gruyere" `Alondo is working to develop the diplomatic core for the Free Worlds. Due to a lack of serious appointed positions (with the few already existing occupied as ambassadors to the Republic), he is looking after the Free World's interests during this crisis.`
+		conversation
+			`When you land on Deep Katya is there waiting for you along with Alondo, "<first>! It's so good to see you again! When the Republic briefed the Chancellor, he brought us in and we were very concerned by the lack of ability to reach you, but of course you would somehow manage to be in the thick of it before we even knew what was going on."`
```

Comment:
These titles are never referred to yet at any point. This story introduces them officially for the first time.

Reply:
My concern here is that a Chancellor is never even mentioned during the entirety of the Free World's storyline, which specifically involves interacting with the Republic gov. If there was a Chancellor, it seems shocking that we would have never heard it even mentioned during Free Worlds. If the Chancellor is so integral to the story here that it doesn't work without their existence. I'd argue it needs to be written into the existing FW story in at least some form or fashion to avoid it feeling like a retcon.

Reply:
I would counter that, unless executive power is vested in the head role (like with the President of the US), it's unlikely that any of the actual *work* involved in the FW plot would involve that leader on either side, over and above the roles of the government or the war ministry in general. The FW equivalent also would likely not have been elected until after the FW story and the council's disbandment.
Whereas a diplomatic crisis with a wholly foreign entity is likely going to significantly revolve around a face character in the media, even if the work still largely happens behind the scenes. In FW your involvement is with the active leadership (the council) and is focused on the real work. In this you're thrust into the limelight of media management.